frontend/src/index.tsx

```diff
@@ -38,12 +39,13 @@ const link = createHttpLink({
 
 const authLink = setContext(async (_, { headers }) => {
   // get the authentication token from local storage if it exists
-  let token: string = getLocalStorageObjProperty(
-    AUTHENTICATED_USER_KEY,
-    "accessToken",
-  );
+  let token: string | null = getLocalStorageObjProperty<
+    NonNullable<AuthenticatedUser>,
+    string
+  >(AUTHENTICATED_USER_KEY, "accessToken");
 
   if (token) {
+    // TODO: define expected JWT return type
     const decodedToken: any = jwt.decode(token);
```

Comment:
@sherryhli @chenrichard10 @zm2he 
Same here, (if it's diff from the other one).

Reply:
It's the same as above